src/runtime_src/core/include/unittests/xrt_h_c_compilation/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0                                           
+# Copyright (C) 2022, Advanced Micro Devices, Inc. All rights reserved.
+
+add_executable(main main.c)
+target_include_directories(main
+  PRIVATE
+  ${CMAKE_CURRENT_SOURCE_DIR}/../../
+  )
```

Comment:
```suggestion
```
Remove, the default include search path when building XRT has `<PATH>/src/runtime_src`